web/src/features/fireWeather/components/tables/NoonForecastTable.tsx

```diff
@@ -143,17 +148,20 @@ const NoonForecastTable = (props: NoonForecastTableProps) => {
                     </TableSortLabel>
                   </TableCell>
                   {/* Temperature */}
-                  <TableCell className={classes.darkColumnHeader}>Forecast</TableCell>
-                  <TableCell className={classes.darkColumnHeader}>Observed</TableCell>
+                  <TableCell className={classes.darkColumnHeader}>FCST</TableCell>
```

Comment:
Why are these being shortened here?

Reply:
Because I split speed and direction & because I've moved the unit of measure into the headings, the tables have gotten so wide they were causing tests to fail.